Controller/ConnectController.php

```diff
@@ -359,4 +348,60 @@ protected function getTemplatingEngine()
     {
         return $this->container->getParameter('hwi_oauth.templating.engine');
     }
+
+    /**
+     * {@inheritdoc}
+     *
+     * Symfony <2.6 BC. To be removed.
+     */
+    protected function isGranted($attributes, $object = null)
+    {
+        if (method_exists('Symfony\Bundle\FrameworkBundle\Controller\Controller', 'isGranted')) {
```

Comment:
hmm, not really helpful as you would still have to choose the right implementation based on the version though for this one